src/pybind/mgr/dashboard_v2/components/rgw_client.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
```

Comment:
why did you create a new directory `components`? We already have the `services` directory to hold this kind of code

Reply:
My intention was to separate code which uses the global MGR instance from those code who doesn't (and shall not) use it. But that changed while developing the proxy, so that `rgw_client.py` does not depend on on the `mgr` instance. I'm going to change that. What do you think regarding the leftover component `rest_client.py`? Shall that stay there or would you also move it the services directory, although it doesn't (and shall not) depend on the mgr instance?

Reply:
I would put `rest_client.py` in the "root" dir, along side with tools.py and module.py

Reply:
Okay, will do so, thanks for the feedback!